augmentedreality/README.md

```diff
@@ -62,6 +62,10 @@ Android 7.0 64-bit (armv8) is the minimum requirement. 32-bit (armv7) is not sup
 
 The latest ArcGIS Maps SDK for Qt is required to use this toolkit component.
 
+The system requirements for the ArcGIS Maps SDK for Qt takes precendent over the minimum versions mentioned here.
```

Comment:
A few lines above this one we mention some very old iOS and Android OSes. Probably the first ones to ever support AR, which is much older than our system requirements mention now. 

I didn't want to change those and have more maintenance, so hopefully this statement will make it clear which document wins.

cc @ldanzinger